pages/dev/guides/example-nft-linker.md

```diff
@@ -1,6 +1,6 @@
 # Example: NFT linker
 
-The [axelar-local-gmp-examples](https://github.com/axelarnetwork/axelar-local-gmp-examples) repo contains an example, [NFT linker](https://github.com/axelarnetwork/axelar-local-gmp-examples/tree/main/examples/nft-linker), covering cross-chain transfers of ERC-721 non-fungible tokens.
+The [axelar-examples](https://github.com/axelarnetwork/axelar-examples) repo contains an example, [NFT linker](https://github.com/axelarnetwork/axelar-examples/tree/main/examples/nft-linker), covering cross-chain transfers of ERC-721 non-fungible tokens.
```

Comment:
`https://github.com/axelarnetwork/axelar-examples/tree/main/examples/nft-linker` to `https://github.com/axelarnetwork/axelar-examples/tree/main/examples/evm/nft-linker`